enterprise/backend/src/metabase_enterprise/search/scoring.clj

```diff
@@ -18,16 +41,25 @@
     1
     0))
 
+(def ^:private legacy-scorers
+  {:official-collection official-collection-score
+   :verified            verified-score})
+
+(defn- legacy-name [k]
+  (if (= k :official-collection)
+    "official collection score"
+    (name k)))
```

Comment:
I don't think the names are used anywhere, but easy enough to preserve for now, so just kept it.